pkgs/by-name/bu/buildstream/package.nix

```diff
@@ -8,6 +8,7 @@
   fuse3,
   lzip,
   patch,
+  protobuf_27,
```

Comment:
It looks like this isn't building:

```
nix-build . -A protobuf_27

... cmake stuff

-- Configuring done (1.7s)
CMake Error at cmake/libprotobuf-lite.cmake:30 (target_link_libraries):
  Target "libprotobuf-lite" links to:

    absl::if_constexpr

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  CMakeLists.txt:303 (include)
```

https://logs.ofborg.org/?key=nixos/nixpkgs.421920&attempt_id=3102e791-f5ea-40d3-87c1-3990d1bed2f1

Reply:
I think is due to me overriding Protobuf. It's best I wait for Buildstream 2.6.0 to be released, where I wouldn't have to do this anymore.

Reply:
Yeah if I add https://github.com/apache/buildstream/commit/9c4378ab2ec71b6b79ef90ee4bd950dd709a0310 and https://github.com/apache/buildstream/commit/456a464b2581c52cad2b0b48596f5c19ad1db23f as patches and drop the `protobuf_27` override 2.5.0 builds fine, so probably worth waiting for a new release

Reply:
I could add those patches in the interim, but I believe the Buildstream folks are planning to release 2.6.0 soon.